README.md

```diff
@@ -8,13 +8,13 @@ OCA banking payment addons for Odoo
 On version 11.0, this project focus on payment interface. The indentation below 
 indicates the dependency graph of the main modules.
 
-- account_banking_payment_export - Basic export functionality of payment orders
+-  `account_banking_pain_base` - Basic export functionality of payment orders (depends on `account_payment_order` & `account_payment_partner`)
```

Comment:
It should be `account_payment_order`

Reply:
Uh.. The 'base' functionality of the SEPA modules are 'account_banking_pain_base', right? Not account_payment_order, besides the fact that account_banking_pain_base is also depending on account_payment_order, what I wrote between brackets.

Reply:
But this is not for SEPA, but for making payment orders for banks. American banks don't use SEPA for example.